src/api/services/iuf/sessions_test.go

```diff
@@ -121,6 +143,21 @@ func TestGetDagTasks(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	defer ctrl.Finish()
 	wfServiceClientMock := &workflowmocks.WorkflowServiceClient{}
+	wfTemplateServiceClientMock := &workflowtemplatemocks.WorkflowTemplateServiceClient{}
+	wt1 := v1alpha1.WorkflowTemplate{}
+	wt1.Name = "this_is_an_operationr_1"
```

Comment:
Are all the instances of `operationr` a typo, or is it intentional?

Reply:
They are all typos 😆

Reply:
Fixed